tcms/telemetry/static/telemetry/css/testing/execution-trends.css

```diff
@@ -0,0 +1,4 @@
+/* this will show the tooltip, but not the actual data point */
```

Comment:
`TOTAL` is shown in the legend below the chart. Remove/hide it b/c you can click on it and it toggles the TOTAL line from the graph but because the line isn't visible it looks like nothing is happening.

Reply:
done.